src/ui/handler/scroll_zoom.js

```diff
@@ -356,6 +377,50 @@ class ScrollZoomHandler {
     reset() {
         this._active = false;
     }
+
+    /**
+     * Returns the scroll zoom blocker alert's HTML element.
+     *
+     * @example
+     * // Change the scroll zoom blocker element's font size
+     * map.scrollZoom.getScrollZoomBlockerElement();
+     * scrollZoomBlockerElem.style.fontSize = "25px";
+     * @returns {HTMLElement} Returns container element.
+     */
+
+    getScrollZoomBlockerElement() {
+        if (this._container) {
+            return this._container;
+        }
+    }
+
+    _addScrollZoomBlocker() {
+        if (!this._map) { return; }
+        if (!this._container) {
```

Comment:
Nit: early return and wrapping if can be combined, e.g.:
```suggestion
        if (this._map && !this._container) {
```